README.md

```diff
@@ -5,8 +5,21 @@
 
 The DIP Metadata Viewer is web-based, interactive table for viewing and filtering the metadata for data sets available through the [Data Innovation Program (DIP)](https://www2.gov.bc.ca/gov/content/data/about-data-management/data-innovation-program)&mdash;a data integration and analytics program for the Government of British Columbia. 
 
-The dip-metadata-viewer is built using the [{flexdashboard}](https://rmarkdown.rstudio.com/flexdashboard/), [{crosstalk}](https://rstudio.github.io/crosstalk/index.html)  and [{reactable}](https://glin.github.io/reactable/) R packages. All metadata is sourced from the [B.C. Data Catalogue](https://catalogue.data.gov.bc.ca/group/data-innovation-program), licensed [Access Only](https://www2.gov.bc.ca/gov/content?id=1AAACC9C65754E4D89A118B875E0FBDA), using the [{bcdata}](https://bcgov.github.io/bcdata/) R package.
+The dip-metadata-viewer is built using the [{flexdashboard}](https://rmarkdown.rstudio.com/flexdashboard/), [{crosstalk}](https://rstudio.github.io/crosstalk/index.html)  and [{reactable}](https://glin.github.io/reactable/) R packages. All metadata is sourced from the [B.C. Data Catalogue](https://catalogue.data.gov.bc.ca/group/data-innovation-program) using the [{bcdata}](https://bcgov.github.io/bcdata/) R package. All the data described in the dip-metadata-viewer is licensed [Access Only](https://www2.gov.bc.ca/gov/content?id=1AAACC9C65754E4D89A118B875E0FBDA) and is only available through the [Data Innovation Program](https://www2.gov.bc.ca/gov/content/data/about-data-management/data-innovation-program)&mdash;the data is _not_ downloadable from the B.C. Data Catalogue.
 
+### Code
+There are two scripts that are required to generate the viewer, they need to be run in order:
+
+- `get-data.R`
+- `dip-metadata-viewer.Rmd`
```

Comment:
For the non-heavy R user, it's more than just _running_ the Rmd file right - you have to knit it to an html?

Reply:
Yes, good catch. I'll edit the `README` to be more specific.